aten/src/ATen/gen.py

```diff
@@ -249,7 +249,7 @@ def format_yaml(data):
     noalias_dumper.ignore_aliases = lambda self, data: True
     # Support serializing OrderedDict
     noalias_dumper.add_representer(OrderedDict, dict_representer)
-    return yaml.dump(data, default_flow_style=False, Dumper=noalias_dumper)
+    return yaml.dump(data, default_flow_style=False, Dumper=noalias_dumper, width=2048)
```

Comment:
Could we replace this with float('Inf') so that we don't have a magic length at which point your yaml parser breaks again? Also, a comment explaining that we turned off line breaks for portability.